catalog/ui/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -502,29 +487,22 @@ define([
             var eventToDrawShape = null;
             var attrsToSet = null;
 
-            switch(data.type) {
-                case "bbox": {
-                    var { north, south, west, east } = data;
-                    attrsToSet = { north, south, west, east };
-                    eventToDrawShape = 'search:bboxdisplay';                    
+            var geometry = data.geometry || {};
+            switch(geometry.type) {
+                case "Polygon": {
+                    var polygon = geometry.coordinates[0]; // outer ring only
+                    attrsToSet = { polygon, bbox: undefined };
+                    eventToDrawShape = 'search:polydisplay';
                     break;
                 }
-                case "polygon": {
-                    var polygon = data.coordinates.map(function(coord){
-                        return [coord.latitude, coord.longitude];
+                case "MultiPolygon": {
+                    var polygon = geometry.coordinates.map(function(ring){
+                        return ring[0]; // outer ring only
                     });
                     attrsToSet = { polygon, bbox: undefined };
                     eventToDrawShape = 'search:polydisplay';
                     break;
                 }
-                case "point-radius": {
```

Comment:
Did we end up not needing the `point-radius` and `bbox` cases anymore?

Reply:
These aren't need for the geocoder; it looks like point-radius won't be used any more and BBOX is converted to `Polygon`